examples/25_weave_render/src/entries.tsx

```diff
@@ -0,0 +1,36 @@
+import { lazy } from 'react';
+import { createPages } from 'waku';
+import type { PathsForPages } from 'waku/router';
+import FooPage from './components/FooPage';
+
+// The use of `lazy` is optional and you can use import statements too.
+const HomeLayout = lazy(() => import('./components/HomeLayout'));
+const HomePage = lazy(() => import('./components/HomePage'));
+
+const pages = createPages(async ({ createPage, createLayout }) => [
+  createLayout({
+    render: 'dynamic',
+    path: '/',
+    component: HomeLayout,
+  }),
+
+  createPage({
+    render: 'static',
+    path: '/',
+    component: HomePage,
+  }),
+
+  createPage({
+    render: 'static',
+    path: '/foo',
+    component: FooPage,
+  }),
+]);
```

Comment:
How about adding another route like `/nested/bar` which has static layout and dynamic page?

Reply:
added... there is weird behavior though

When I nav to /nested/bar then click the link to go back to home or foo, I get a white screen with no content. Is this a bug? Or did I make a mistake?

Reply:
Does it happen both in DEV and PRD? Can be a bug.

Reply:
Wait, `/nested/bar` still has a dynamic root layout?

I think we should make:
- `/foo`: static layout + dynamic page
- `/nested/bar`: static layout + dynamic layout + static page

what do you think?

Reply:
> Does it happen both in DEV and PRD? Can be a bug.

Reply:
> Does it happen both in DEV and PRD? Can be a bug.
Yes it happens both

Reply:
> static layout + dynamic layout + static page

How do I make two layouts on a route?

Reply:
ok, I changed it. let me know if that's what you meant though. I'm not super confident with it yet.

Reply:
It looks different. Okay, we may not need `nested`. How about this?

```js
  createLayout({
    render: 'static',
    path: '/',
    component: HomeLayout,
  }),

  createPage({
    render: 'static',
    path: '/',
    component: HomePage,
  }),

  createLayout({
    render: 'static',
    path: '/foo',
    component: FooLayout,
  }),

  createPage({
    render: 'dynamic',
    path: '/foo',
    component: FooPage,
  }),

  createLayout({
    render: 'dynamic',
    path: '/bar',
    component: BarLayout,
  }),

  createPage({
    render: 'static',
    path: '/bar',
    component: BarPage,
  }),

  ```

Reply:
yeah, makes sense ! lgtm - changed